spyder/preferences/languageserver.py

```diff
@@ -608,30 +610,70 @@ def mouseDoubleClickEvent(self, event):
 class LSPManagerConfigPage(GeneralConfigPage):
     """Language Server Protocol manager preferences."""
     CONF_SECTION = 'lsp-server'
-    NAME = _('Language Server Protocol')
+    NAME = _('Code completion and linting')
```

Comment:
We should make this shorter, e.g. just ``Code Analysis`` or come up with our own umbrella name for completion, linting, introspection, etc. like Microsoft did (I vote for ``Spyder Sense`` :) ) Otherwise, it makes the sidebar way too wide and leaves less space for the rest of the preferences. (I also plan to shorten ``Current working directory`` to ``Working directory`` in my UI text pass when Spyder 4 is near-final).

Reply:
I understand what you mean but I think it's better to be clear and explicit with the section name so that people understands what it stands for. `Code Analysis`, while short, is extremely vague and hard to grasp in my opinion.

Reply:
Right; it is important to be clear and explicit with the name. What about ``Completion and linting? Is that clear enough?

Aside from that, @jnsebgosselin @jitseniesen any suggestions on a clear umbrella term for completion/introspection/linting (real-time-analysis) functionality? Or are we stuck with enumerating them every time (that has been an issue in other contexts too).

Reply:
> Completion and linting

I like this suggestion :+1: